src/dialects/oracle/schema/trigger.js

```diff
@@ -59,6 +60,7 @@ const trigger = {
       `  FROM all_constraints cons, all_cons_columns cols` +
```

Comment:
Same comment as above, using `user_constraints` and `user_cons_columns` might be a better option.